great-idea/services.html

```diff
@@ -16,39 +16,40 @@
 
   <!-- My changes start here -->
   <div class="container-page">
-      <div class="container-header">
-        <div class="box-nav">
-            <body>
-              <a href="">Services</a>
-              <a href="">Product</a>
-              <a href="">Vision</a>
-              <a href="">Features</a>
-              <a href="">About</a>
-              <a href="">Contact</a>
-              <a href="">
-                  <img class="logo" src="index.html" alt="Great Idea! Company logo.">
-                  </a>
-                <!-- original nav content:
-                  <body>
-                  Services
-                  Product
-                  Vision
-                  Features
-                  About
-                  Contact -->
-              </body>
-          </div>
-          <!-- <div class="box-logo"> 
+    <div class="container-header">
+      <div class="box-nav">
+        <nav class="box-nav">
+
```

Comment:
because you now have more than one page you should probably have a way to navigate between them. Fill in a couple of those href values.